format/index.md

```diff
@@ -855,6 +855,18 @@ A server **MUST** prepare responses, and a client **MUST** interpret
 responses, in accordance with
 [`HTTP semantics`](http://tools.ietf.org/html/rfc7231).
 
+### Errors
+
+A server **MAY** choose to stop processing operations as soon a problem is
+encountered, or it **MAY** continue processing operations and encounter
```

Comment:
perhaps remove "operations" here - that really just applies to jsonpatch operations